resources/js/components/utils/object_merge.ts

```diff
@@ -0,0 +1,15 @@
+export function objectMerge<T>(...objects: (Partial<T> | undefined)[]): T {
```

Comment:
Przydałby się jakiś komentarz z dokumentacją i może przykładem, ew. zmiana nazwy funkcji na objectsRecursiveMerge - bo to jest chyba główna idea.

Reply:
Zmieniłem nazwę. No i są testy jak ktoś chce zobaczyć przykłady.